spec/restaurant_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'restaurant'
```

Comment:
There's some guidance on this is the review rubric
https://github.com/ajosephides/takeaway-challenge/blob/master/docs/review.md#stubbing-the-twilio-api